deepchem/dock/pose_generation.py

```diff
@@ -337,7 +284,7 @@ def generate_poses(self,
       A numpy array of shape `(3,)` holding the size of the box to dock. If not
       specified is set to size of molecular complex plus 5 angstroms.
     exhaustiveness: int, optional (default 10)
-      Tells Autodock Vina how exhaustive it should be with pose
+      Tells Autodock Vina the number of MC runs it should run for pose
```

Comment:
I'd keep the original general explanation (but optionally add on the new explanation) since most people won't know MC = Monte Carlo and what that means